cmd/cluster-init/testdata/ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -0,0 +1,45 @@
+periodics:
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
+    - base_ref: master
+      org: openshift
+      repo: release
+  interval: 12h
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-openshift-release-master-existingCluster-apply
+  spec:
+    containers:
+      - args:
+          - --config-dir=clusters/build-clusters/exisingCluster
+          - --confirm=true
+          - --as=
+          - --kubeconfig=/etc/build-farm-credentials/kubeconfig
+        command:
+          - applyconfig
+        env:
+          - name: github_client_id
```

Comment:
I think we could remove this for now.
When we have a decision on https://issues.redhat.com/browse/DPTP-2497, we could adjust accordingly.

Reply:
actually, I don't even need this particular file in the testdata anymore now that I am no longer checking for existence of a particular periodic.

Reply:
My bad, that is from an existing job.
Please ignore the above comment.

Reply:
Well at least you reminded me that I didn't need this file anymore ;)